main.go

```diff
@@ -1,38 +1,44 @@
 package main
```

Comment:
The structure of the folders and files in your service is incorrect, this logic shouldn't suppose to be in main.go